app/controllers/performances_controller.rb

```diff
@@ -17,11 +17,43 @@ def show
 
   def new
     if current_user.actor?
-      @user = current_user
-      authorize @performance = Performance.new(user: @user)
+      authorize @performance = Performance.new
     end
   end
 
+  def create
+    if current_user.actor?
+      @character_category = CharacterCategory.find_by_name(performance_params[:character_category])
+      if @character_category.nil?
```

Comment:
T'as une méthode qui te simplifierais ta vie
C'est un truc du style
```suggestion
      #find_or_create
```
Voir la doc

https://apidock.com/rails/v4.0.2/ActiveRecord/Relation/find_or_create_by

Reply:
cool ca a l'air intéressant, ok j'integre